cypress/e2e/ProductionSmokeTests/$SBIR/productionTests.js

```diff
@@ -39,4 +39,49 @@ And('page title is {string}', (title) => {
 
 And('the text {string} is displayed', (str) => {
     cy.get(`p:contains("${str}")`).should('be.visible');
+});
+
+Then('all displayed events have a future date', () => {
+    const months = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ];
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = date.getMonth();
+    const day = date.getDate();
```

Comment:
When working with parts of multiple dates, these names become ambiguous and easily confused. It's more typing, but `currYear`, `currMonth`, and `currDay` are clearer.

Reply:
Agreed ( I am bad at naming) - renamed!